src/backend/access/external/test/pxffilters_test.c

```diff
@@ -61,7 +62,7 @@ test__supported_filter_type(void **state)
 
 	/* go over pxf_supported_types array */
 	int nargs = sizeof(pxf_supported_types) / sizeof(Oid);
-	assert_int_equal(nargs, 12);
+	assert_int_equal(nargs, 13);
```

Comment:
This test is to check pxf_supported_types number, it's old value is hard-coded too(be 12), after an increase of DATEOID become 13.